.github/workflows/main.yml

```diff
@@ -20,7 +20,7 @@ concurrency:
 
 env:
   IDAES_CONDA_ENV_NAME_DEV: examples-pse-dev
-  IDAES_PSE_REQUIREMENT: idaes-pse[prerelease] @ https://github.com/IDAES/idaes-pse/archive/main.zip
+  IDAES_PSE_REQUIREMENT: idaes-pse[prerelease] @ git+https://github.com/IDAES/idaes-pse@refs/pull/978/head
```

Comment:
Just a reminder to switch this back to the original line before merging this PR.